adapters/jdbc/src/main/java/org/atomhopper/jdbc/adapter/JdbcFeedSource.java

```diff
@@ -348,20 +357,39 @@ private Feed hydrateFeed(Abdera abdera, List<PersistedEntry> persistedEntries,
 
     private Entry hydrateEntry(PersistedEntry persistedEntry, Abdera abderaReference) {
 
-        final Document<Entry> hydratedEntryDocument = abderaReference.getParser().parse(
-              new StringReader( persistedEntry.getEntryBody() ) );
+//        final Document<Entry> hydratedEntryDocument = abderaReference.getParser().parse(
+//              new StringReader( persistedEntry.getEntryBody() ) );
+        String entryBody = persistedEntry.getEntryBody();
```

Comment:
Why are we holding a String when we want the Document at the end? Just call the method in the StringReader on line 377.